addons/mail/static/src/model/model_manager.js

```diff
@@ -417,7 +417,7 @@ export class ModelManager {
      * @param {Object} [data={}]
      */
     _addDefaultData(Model, data = {}) {
-        const data2 = {};
+        const data2 = { ...data };
         for (const field of Model.__fieldList) {
             // `undefined` should have the same effect as not passing the field
             if (data[field.fieldName] !== undefined) {
```

Comment:
Isn't the first condition what spread does now?

Reply:
It spreads on model fields but not on `data`, so resulting `data2` was only data on actual fields.
This was problematic because check happens after call of `_addDefaultData`.

Reply:
But having spread now strictly does more than manually adding fields like before, so this line becomes useless.